ndb/src/google/cloud/ndb/__init__.py

```diff
@@ -126,6 +127,7 @@
 from google.cloud.ndb.context import ContextOptions
 from google.cloud.ndb.context import EVENTUAL_CONSISTENCY
 from google.cloud.ndb.context import TransactionOptions
+from google.cloud.ndb._eventloop import async_context
```

Comment:
Are you going for alphabetical or something else?

Reply:
heh, 'LC_ALL=C'. )

Reply:
Yes, alphabetical.  Human alphabetical, not necessarily what `sort()` would do.